### PR TITLE
Stop gentle

### PR DESCRIPTION
**Pull request by therbun**
added stop gentle

### CODE REVIEW

File: src/main/java/org/bitbuckets/drive/DriveSubsystem.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ void handleLogic() {
 
 
         if (nextStateShould == DriveFSM.IDLE) {
-            driveControl.stop();
+            driveControl.stopGentle();
```
Comment (DanielWhatsGood) [inline src/main/java/org/bitbuckets/drive/DriveSubsystem.java:155]: stopGentle should be used in teleop

File: src/main/java/org/bitbuckets/drive/DriveSubsystem.java [hunk 2]
```diff
@@ -214,7 +214,7 @@ void teleopNormal() {
         switch (orientation.readValue()) {
             case FIELD_ORIENTED:
                 if (xOutput == 0 && yOutput == 0 && rotationOutput == 0) {
-                    driveControl.stop();
+                    driveControl.stopGentle();
```
Comment (DanielWhatsGood) [inline src/main/java/org/bitbuckets/drive/DriveSubsystem.java:217]: stopGentle should be used in teleop

File: src/main/java/org/bitbuckets/drive/DriveSubsystem.java [hunk 3]
```diff
@@ -226,7 +226,7 @@ void teleopNormal() {
                 break;
             case ROBOT_ORIENTED:
                 if (xOutput == 0 && yOutput == 0 && rotationOutput == 0) {
-                    driveControl.stop();
+                    driveControl.stopGentle();
```
Comment (DanielWhatsGood) [inline src/main/java/org/bitbuckets/drive/DriveSubsystem.java:229]: stopGentle should be used in teleop

File: src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java [hunk 4]
```diff
@@ -17,7 +17,8 @@
 /**
  * Represents a real drive controller that implements control of the drivetrain using a list of SwerveModule interfaces
  */
-public class DriveControl implements IDriveControl, HasLogLoop {
```
Comment (DanielWhatsGood) [inline src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java:20]: this is correct

File: src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java [hunk 5]
```diff
@@ -86,7 +87,14 @@ public void stopSticky() {
         });
     }
 
-    
+    public void stopGentle() {
```
Comment (DanielWhatsGood) [inline src/main/java/org/bitbuckets/drive/controlsds/DriveControl.java:90]: we want this

**Review by DanielWhatsGood — APPROVED** [review]

**Review by Fireye04 — APPROVED** [review]

**Review by DanielWhatsGood — APPROVED** [review]

**Pull request merged**